release/RELEASE.md

```diff
@@ -147,32 +147,60 @@ how to communicate around it and when to remove the `prerelease` marker on the
 
 ## Testing
 
-This testing procedure starts once the release is listed on the [releases page].
+This testing procedure starts once the release is listed on the [releases page]. 
 
-1. On Windows, install the new SDK using the installer on the [releases page].
+In the following notes, we assume that `$VERSION` contains
+the full version tag for the release you are testing - in other words, the full version as recorded on the Slack 
+`#team-daml` message that is generated when closing the `main` build PR.
 
-   On macOS/Linux:
+For example, for the Slack message:
+> team_daml_notifs
+>
+> Just published `2.4.0-snapshot.20220830.10494.0.4622de48`.
+>
+> For testing:
+>
+> ...
```

Comment:
If we're going to reference the Slack message, should we also note that it includes the (expanded) Windows link and the (expanded) curl command?